README.md

```diff
@@ -107,7 +131,13 @@ yarn start --sources packages/esm-form-engine-app --backend https://link-to-my-b
 ```
 
 ### Production
-To use the form engine as your default in your O3 instance, you will need to add this to your importmap.
+
+Note that in addition to this library, other form engine alternatives exist within the O3 ecosystem, including:
+
+- [The HTML Form Entry module](https://github.com/openmrs/openmrs-module-htmlformentry)
+- [The Angular Form Engine](https://github.com/openmrs/openmrs-ngx-formentry)
+
+To use this library as the default form engine in your O3 instance, you will need to add this to your importmap:
```

Comment:
Is it fair to assert that RFE is currently the default FE lib for O3?

Reply:
Fixed in the latest commits.